src/lib/device-types/index.ts

```diff
@@ -125,43 +125,40 @@ function fetchDeviceTypes(): Promise<Dictionary<DeviceTypeInfo>> {
 	const result: Dictionary<DeviceTypeInfo> = {};
 	getIsIgnored.clear();
 	getDeviceTypeJson.clear();
-	return listFolders(IMAGE_STORAGE_PREFIX)
-		.map(slug => {
-			return listFolders(getImageKey(slug))
-				.then(builds => {
-					if (_.isEmpty(builds)) {
+	return Promise.map(listFolders(IMAGE_STORAGE_PREFIX), slug => {
+		return listFolders(getImageKey(slug))
+			.then(builds => {
```

Comment:
How come you didn't go all the way converting this one?

Reply:
This was just making the minimal changes here to accomodate the `src/lib/device-types/storage/s3.ts` conversion, I'll eventually convert this file as well I'm sure